### ISSUE TITLE
Would it doable to implement Cassandra as database?

### ISSUE THREAD

**Issue by Radek-Suski** [traccar/traccar#4962]
**Is your feature request related to a problem? Please describe.**
No
**Describe the solution you'd like**
Possibility to use Cassandra, ideally AWS spaces

**Describe alternatives you've considered**
Other standard databases. These are mostly hard to scale vertically 

**Additional context**
Nothing

**Comment by tananaev** [comment]
It should be doable to implement support, but it's not going to be as simple as supporting another SQL database.

**Comment by Radek-Suski** [comment]
Yeah I assumed it. I’m gonna take a look. I didn’t do anything in Java since I left university though. 

Thanks a lot for your answer

**Comment by tananaev** [comment]
If you want to give it a try, check the `Storage` interface.

**Comment by Radek-Suski** [comment]
Thanks for the hint. Should I be able to achieve this I’d of course back-contribute. 

And thanks for your amazing work

**Comment by tananaev** [comment]
It looks like there's no a lot of interest in this and Cassandra's popularity is slowly declining. Closing as there's no plan to implement official support for it for now.
